.github/workflows/prod-build-deploy-cloudflare.yml

```diff
@@ -0,0 +1,70 @@
+name: Prod Build and Deploy (CloudFlare)
+
+on:
+  push:
+    branches: ['main']
+  workflow_dispatch:
+  pull_request:
+
+env:
+  GOOGLE_ANALYTICS_TRACKING_ID: ${{ secrets.GOOGLE_TAGMANAGER_ID_SCDF }}
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  # build and deploy staging
```

Comment:
**A couple of infos:**
- This comment says we are deploying into `staging`
- The workflow file is named `prod-build-*`
- The output generates a preview link (I am guessing this is staging)
- We have the commented out blue/green deploy action below

**A couple of questions:**

1. What is the output of this PR before the merge.
2. What is the output once the PR is merged?
3. How do we get to production?